src/Async/Qu.swift

```diff
@@ -0,0 +1,200 @@
+//
+//  Qu.swift
+//
+//  Created by Logan Wright on 10/24/15.
+//  Copyright © 2015 lowriDevs. All rights reserved.
+//
+
+public typealias Block = () -> ()
```

Comment:
Cool. So if we start including a new library that is also called block, all our existing references for that project will have to be renamed, something like that?